src/components/Celebration.js

```diff
@@ -1,10 +1,33 @@
-import "./Celebration.css"
+import "./Celebration.css";
+import React, { useRef } from 'react';
+import html2canvas from 'html2canvas';
 
 export default function Celebration({highscore, time}){
+    const highScoreRef = useRef(null);
+
+  const takeScreenshot = () => {
+    if (!highScoreRef.current) {
+      console.error("Element not found.");
+      return;
+    }
+
+    html2canvas(highScoreRef.current, { useCORS: true , backgroundColor:"black"}).then((canvas) => {
+      const image = canvas.toDataURL("image/jpeg");
+      const a = document.createElement("a");
+      a.href = image;
+      a.download = "high_score_screenshot.jpeg";
+      a.click();
+    }).catch((error) => {
+      console.error("Error capturing screenshot:", error);
+    });
+  };
    return (
-    <div className="celebration-container">
-        <div className="celebration-notification">
-            <p>Highscore achieved: <span>{highscore}</span><br/>Time Taken: <span>{time}s</span></p>
+    <div className="celebration-container" >
+        <div className="celebration-notification" >
+            <p ref={highScoreRef}> <h3>A&A Match</h3>Highscore achieved: <span>{highscore}</span><br/>Time Taken: <span>{time}s</span></p>
```

Comment:
I don't like the fact about seeing to time the title here.

Can you maybe make this screenshot (without the second title): 
![grafik](https://github.com/idna001/Couple-Memory/assets/46223359/194f51e6-1f86-491b-af45-72b5fda76b5f)

Reply:
Yeah, should I just remove time from the screenshot or from the highscore alert?

Reply:
I mean removing the second title, that you added in the banner. 
If it's possible, It would be great if you can see not only the banner in the Screenshot. 

Better would be to see also the pictures in the background and the above title. 

But that are only little improvements. It's okay without the second title.